Dockerfile

```diff
@@ -1,49 +1,63 @@
-FROM python:3.8-slim-buster
-EXPOSE 5000
-LABEL maintainer badsmoke "dockerhub@badcloud.eu"
+ARG PYTHON_IMAGE_TAG=2.7-slim-buster
+
+FROM buildpack-deps:curl AS ffmpeg
+RUN apt-get update && apt-get install -y xz-utils
+RUN curl -fsSLO https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-i686-static.tar.xz \
```

Comment:
I never worked with multi-arch image, but how do you plan to detect the architecture and install the right ffmpeg ?

I would mind having scripts instead of having everything in one Dockerfile. I guess it can help understanding this file to have just one line like this
```sh
RUN ./install-ffmpeg.sh
 ```

Reply:
I'd say that's an issue for another PR, and shouldn't be in the scope of this one (since the existing build isn't multi-arch, and the scope of this PR is adding multi-stage builds), however i imagine something like this: 

```
RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
    && case "${dpkgArch##*-}" in \
      amd64) ARCH='x64';; \
      ppc64el) ARCH='ppc64le';; \
      s390x) ARCH='s390x';; \
      arm64) ARCH='arm64';; \
      armhf) ARCH='armv7l';; \
      i386) ARCH='x86';; \
      *) echo "unsupported architecture"; exit 1 ;; \
    esac \
RUN curl -fsSLO https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-$ARCH-static.tar.xz
```